hawc/services/epa/dsstox.py

```diff
@@ -34,32 +30,15 @@ def create_from_dtxsid(cls, dtxsid: str) -> "DssSubstance":
         if not re.compile(RE_DTXSID).fullmatch(dtxsid):
             raise ValueError(f"Invalid DTXSID: {dtxsid}")
 
-        obj = cls.create_from_identifier(dtxsid)
-
-        if obj.dtxsid != dtxsid:
+        response = requests.get(
+            f"https://comptox.epa.gov/dashboard-api/ccdapp2/chemical-detail/search/by-dsstoxsid?id={dtxsid}"
+        )
+        response_dict = response.json()
+        if response_dict == [] or response_dict["dsstoxSubstanceId"] != dtxsid:
             raise ValueError(f"{dtxsid} not found in DSSTox lookup")
 
-        return obj
-
-    @classmethod
-    def create_from_identifier(cls, id_: str) -> "DssSubstance":
```

Comment:
since this endpoint no longer works b/c it uses actor; i just removed it. I did the same in the migrations; no need to keep old code around that will never work again.